Server/db/MongoDB.js

```diff
@@ -32,7 +32,15 @@ const connect = async () => {
  */
 const insertUser = async (req, res) => {
   try {
-    const newUser = new UserModel({ ...req.body });
+    const userData = { ...req.body };
+    console.log(req.file);
+    if (req.file) {
+      userData.profileImage = {
+        data: req.file.buffer,
+        contentType: req.file.mimetype,
+      };
```

Comment:
Was there a reason to include the image upload logic in `insertUser`? If I'm not mistaken this is the function that handles user registration correct?

Reply:
I assume users should be able to select a profile image when they create an account?

Reply:
That would make sense, I was only asking because the figma design plans doesn't include that upload feature.